### PR TITLE
chore(QPT): ANA-1572 Add password as additional param to create-account

### PR DESCRIPTION
**Pull request by avattipalli**
Jira: [ANA-1752](https://jira.bigcommerce.com/browse/ANA-1752)

## What? Why?
Currently, we have hardcoded string as a password in create-account service. We want to remove that and require the service consumers to pass in the password as an additional parameter to email_address and account type.

This way service consumers generate and know their password and we don't have to send them back password or hardcode it or store them as secrets.

## How was it tested?
Updated current tests and ran them locally

- [ ] Cloud Dev end-to-end - Describe Steps
- [ ] Additional test cases (Optional)
- [ ] Integration - Describe Steps (Optional)

## Testing / Proof
...

## How can this be undone?
1. Revert this PR

ping @bigcommerce/billing-iam-testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@wdio/cli": "^6.1.12",
     "@wdio/local-runner": "^5.22.1",
     "@wdio/selenium-standalone-service": "^5.16.10",
-    "bc-store-setup-node": "git+ssh://git@github.com/bigcommerce-labs/bc-store-setup-node.git#master",
+    "bc-store-setup-node": "github:bigcommerce-labs/bc-store-setup-node",
```
Comment (j05h) [inline package.json:23]: Nice, this is cleaner.

File: test/services/create-account-test.js [hunk 2]
```diff
@@ -12,15 +12,15 @@ describe('createAccount', () => {
     createAccountFailureMock();
     expect.assertions(1);
     try {
-      await createAccount('example@example.com', 'non-trial');
+      await createAccount('example@example.com', 'Password1!',  'non-trial');
```
Comment (j05h) [inline test/services/create-account-test.js:15]: We should also use `faker` here.

File: test/services/create-account-test.js [hunk 3]
```diff
@@ -12,15 +12,15 @@ describe('createAccount', () => {
...
     } catch(e) {
       expect(e).toEqual("Account creation failed for example@example.com!")
     }
   });
 
   it('returns the body on success', async () => {
     createAccountBcSetupNodeSuccessMock();
-    const res = await createAccount('example@example.com', 'non-trial');
+    const res = await createAccount('example@example.com', 'Password1!', 'non-trial');
```
Comment (j05h) [inline test/services/create-account-test.js:23]: We should also use `faker` here.

**Review by j05h — CHANGES_REQUESTED** [review]

**Review by j05h — APPROVED** [review]

File: src/services/create-account.js [hunk 4]
```diff
@@ -22,14 +22,14 @@ function _buildCreateAccountPayload(emailAddr) {
       first_name: contactAttributes.first_name,
       last_name: contactAttributes.lastName,
       company_name: faker.company.companyName(),
-      password: Globals.TEST_USER_PASSWORD,
+      password: password,
```
Comment (MarkJCram) [inline src/services/create-account.js:25]: Are we including the passed password as part of the logging/reporting?  Not a huge thing if we're going to creating a new account each time, but if we end up passing a static secret password this wouldn't be ideal

Comment (avattipalli) [reply src/services/create-account.js:25]: Good catch! I was printing it on the logs as part of testing initially but we don't require that. I will remove it.

File: src/services/store-service.js [hunk 5]
```diff
@@ -49,12 +49,11 @@ async function pollForFulfilledProvisioningRequest(requestId, account) {
             throw errorMessage;
         }
     } while (provisioningResponse.data.workflow_state !== 'fulfilled');
-    console.log('******************** Store provisioning is complete ********************\n');
+    console.log('******************** Store provisioning is complete ********************');
     console.log(`Store primary hostname: ${provisioningResponse.data.services[0].primary_hostname}`);
     console.log(`Store current status: ${provisioningResponse.data.services[0].current_state}`);
-    console.log(`Integration login url: ${BMP_HOST}`);
+    console.log(`Login url: ${CP_HOST}`);
```
Comment (MarkJCram) [inline src/services/store-service.js:55]: Good catch on this

**Review by MarkJCram — APPROVED** [review]

**Pull request merged**